techniques/system/distributePolicy/1.0/propagatePromises.st

```diff
@@ -183,6 +183,7 @@ bundle agent sendInventoryToCmdb
     any::
       "incoming_inventory_files" slist => findfiles("${g.rudder_inventories}/incoming/*.ocs", "${g.rudder_inventories}/incoming/*.ocs.gz");
       "updates_inventory_files" slist => findfiles("${g.rudder_inventories}/accepted-nodes-updates/*.ocs", "${g.rudder_inventories}/accepted-nodes-updates/*.ocs.gz");
+      "disable_upload" string => fileexists("/opt/rudder/etc/no_inventory_upload");
```

Comment:
this should be in a classes: section, with the syntax
`"disable_upload" expression => fileexists("/opt/rudder/etc/no_inventory_upload");`

it is used in several bundle, so it should either be in a bundle common, or with a scope namespace

Reply:
oops indeed for the class
but it is used in a single bundle